sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileAsyncApiTests.java

```diff
@@ -919,40 +911,37 @@ public void startCopyLeaseFail() {
     @MethodSource("com.azure.storage.file.share.FileShareTestHelper#startCopyArgumentsSupplier")
     public void startCopyWithOptions(boolean setFilePermissionKey, boolean setFilePermission, boolean ignoreReadOnly,
```

Comment:
You can make the same changes for this test that we made above - removing .defer and .from, moving `String sourceURL = primaryFileAsyncClient.getFileUrl();` outside of the chain, and removing redundant chaining.